README.md

```diff
@@ -19,19 +19,22 @@ This repo is aimed at powering useful applications based on such models on compu
 clip.cpp also has a short startup time compared to large ML frameworks, which makes it suitable for serverless deployments where the cold start is an issue.
 
 ## Hot topics
-
+-   09/27/2023: clip.cpp now uses a new model file structure in GGUF format. This is a breaking change. See bwlow for more information.
 -   09/14/2023: All functions are C-compatible now. `zsl` example is updated to match Huggingface's zero-shot behavior in the zero-shot pipeline.
 -   09/11/2023: Introduce Python bindings.
 -   07/12/2023: Batch inference support for image encoding.
 -   07/11/2023: Semantic image search [example](examples/image-search/README.md) directly in C++.
 
+## BREAKING CHANGE
+### As of 09/27/2023, clip.cpp uses a new model file structure in GGUF format. The latest commit that is compatible with older .bin files is [05f2efd8081b8695e8174dea7a82116ece2fdf63](https://github.com/monatis/clip.cpp/commit/05f2efd8081b8695e8174dea7a82116ece2fdf63). There will be no back compatibility support for older models, and you are recommended to update to the new code and model files. The new structure allows text-only and vision-only model files as well as 32-bit floating point precision, 5-bit and 8-bit quantization support in addition to the existing 16-bit floating point precision and 4-bit quantization.
+
 ## Note about image preprocessing
 
 PIL uses a two-pass convolutions-based bicubic interpolation in resizing with antialiasing applied. In Pytorch, antialiasing is optional. It needs some extra attention to implement this preprocessing logic that matches their results numerically. However, I found that linear interpolation is also good enough for both comparison of different embeddings from this implementation and also comparison of an embedding from this implementation and another one from Transformers. So let's use it until we craft a proper bicubic interpolation.
 
 ## Preconverted Models
 
-Preconverted Models can be found in [HuggingFace Repositories tagged with `clip.cpp`](https://huggingface.co/models?other=clip.cpp).
+Preconverted Models can be found in [HuggingFace Repositories tagged with `clip.cpp-gguf`](https://huggingface.co/models?other=clip.cpp).
```

Comment:
you sure this is a good idea? i can take down /reupload my repos too.

also, you only changed the text, not the link

Reply:
I don't have a strong opinion in this, but I just wanted to close this PR to jump to other tasks (implement BERT and large multimodal models). I only uploaded a few models --my internet speed is not so good. If you'd like to re-upload models I'd only be happy :D and new shell scripts might help you with conversions and quantizations.